SwiftDraft/ReliantFramework/ReliantFrameworkTests/ReliantFrameworkTests.swift

```diff
@@ -54,17 +54,25 @@ struct ContextNeedingContext : ReliantContext {
         return ContextNeedingContext()
     }
 }
+class SubWaver : Waver {
+    func wave(reason: String) -> String {
+        return "Substitute waving"
+    }
+}
 
-
-
-
+class SubstituteContext : SimpleReferenceContext {
+    override init() {
+        super.init()
+        waver = SubWaver()
+    }
+}
 
 class ReliantFrameworkTests: XCTestCase {
     
     override func setUp() {
         super.setUp()
         ReliantFrameworkTestsHelper.sharedInsance.reset()
-        ContextCache.sharedInstance.cache = Dictionary()
+        ContextCache.standard.removeAll()
```

Comment:
See, this I would like to avoid, but I don't know how, since we are tight to the global relyOn function...